docs/jwst/cube_build/reference.rst

```diff
@@ -5,9 +5,9 @@ There are two types of reference files used by the cube_build step. The first ty
 cube parameters used in setting up the output IFU Cube. The reftype for this reference file is *cubepars*
 and there is a  reference file of this type for MIRI data and one for NIRSPEC data. These files contain  tables 
 for each band of the spatial and spectral size and the size of the region of interest to use to construct the 
-IFU cube. For build 7.1, if more than one band is used to build the IFU cube,
+If more than one band is used to build the IFU cube,
```

Comment:
Need to reinsert "IFU cube." at the beginning of this line, in order to finish the sentence from the previous line.